### ISSUE TITLE
Content

### ISSUE THREAD

**Issue by webmaxru** [webmaxru/progressive-web-conf#2]

**Comment by torgo** [comment]
It's vital to ensure the make-up of this conference represents the diverse web developer and creator community. Please take a look at the speaker line-up that we put together for [Samsung Create](https://samsungcreatec.com). See also the speaker line-up for the [DeltaV conference](http://deltavconf.com/) that ran in London last week. Also please note our [diversity & inclusion policy](https://samsunginter.net/diversity-inclusion-statement) for conference participation at Samsung Internet (and my [related blog post](https://medium.com/samsung-internet-dev/updating-our-diversity-inclusion-statement-ac13848faed8)).  In order for Samsung Internet to participate we need to ensure diversity & inclusion is a core principle for this event when it comes to the speaker line-up and all other organizational & logistics issues. See also [Ada's post](https://medium.com/samsung-internet-dev/help-someone-has-pointed-out-my-conference-has-diversity-issues-c1162a1e8d4c?source=collection_home---4------3----------------) on our developer blog for some of our ideas on how to do this.

**Comment by webmaxru** [comment]
I agree with all the points. My next issue on this repo will be "Diversity & inclusion" - looking at the [current participant list](https://github.com/webmaxru/progressive-web-conf/blob/master/PARTICIPANTS.md) - we have an issue here. I invite everyone to help to make this event in a right way from the very beginning.

**Comment by torgo** [comment]
IMO we need to start from scratch on curating the speakers. Accordingly, I have proactively taken all of our Samsung Internet speakers off the list for now.

**Comment by webmaxru** [comment]
https://github.com/webmaxru/progressive-web-conf/issues/6

**Comment by webmaxru** [comment]
CFP is something to discuss with the whole team (when we get someone there :). That's not so regular to have the curated list of speakers, but for me, this is not related to having more or less diverse line-up. The idea is to have the really specific types of sessions (see this issue's description), and there are many great speakers from all the diverse groups who could deliver those. [ColdFront](https://2018.coldfront.co/) conference co-organized by [Kenneth Auchenberg](https://github.com/auchenberg) is the great example of the event with curated content.

**Comment by poshaughnessy** [comment]
I think having sessions from / inviting on-stage people actually building PWAs would be good! Our philosophy with Samsung Create was to involve the community and help showcase other people's great work too, rather than just making it all about us...

**Comment by thisisjofrank** [comment]
Maybe I didn't explain myself well. The point I was trying to make was that (and I realize that this is not your plan, but) filling a lineup with those who have 'expressed interest' or put themselves forward is going to, buy nature, exclude those who don't do that (women and people of colour are less likely to  suggest themselves for speaking slots, for whatever reason). Using a CFP makes it clear to those who are interested in your event that their input is welcome and that they are welcome. 
Even starting from that point is going to leave you with a more skewed lineup. You may not consider it part of the diversity or inclusion of your event, but I'm suggesting that it could/should be considered.
Even with the desired speaking candidates you can put out a CFP stating your preferences. You might be pleasantly surprised by the submissions.

**Comment by webmaxru** [comment]
@poshaughnessy @thisisjofrank thank you for your replies. I love this kind of discussion - lots of good reasoning. Let me elaborate on my proposal then: one of the main (if not main) ideas of having this very niche event is to have the reps from all browser/platform vendors. Otherwise, we can't seriously use "cross-platform" term on this event, and thus there is no reason to hold the event at all. This kind of line-up is easier to organize by inviting the folks, rather than waiting for someone from browser_X will submit CFP (or not).

If we have my above concern addressed - I'm happy with any option, with CFP / by invitation / or mixed (like actually most of the conferences go). This is something for the program committee to decide (not me)

**Comment by poshaughnessy** [comment]
I didn't mean that we couldn't have representatives from each browser, but that it would be nice to _also_ invite other people to present too?

Also I like the idea of having every major browser attend, but I don't really understand why there's "no reason to hold the event at all" if we can't get one browser to commit, for example. The PWA Dev Summit seemed like a success from all I heard, even though not _all_ major browsers were participating publicly, having [Google, Mozilla, Microsoft, Samsung and Opera all on stage together](https://twitter.com/nicoinch/status/745216197004754944) was still pretty great.

**Comment by torgo** [comment]
Hi @webmaxru I think we disagree on this this core point. Having the major browser vendors talk about PWAs is great but having the community of developers also talk about what they are doing with PWAs is even more important.  I think using the Google-run PWA dev summit (which was at https://events.withgoogle.com/progressive-web-app-dev-summit/ but Google have broken the web and removed that link I guess - thanks google) is a good idea. Video of day 1 is here: https://www.youtube.com/watch?v=oujlrIZkyYY I also ran even even on Progressive Web Apps in April 2016 that put a focus on both browsers and developers - see https://webprogressions.org/ . So yes it's important to put browsers on stage showing support. but what is much more interesting is to hear how the community is using these technologies.

**Comment by webmaxru** [comment]
@poshaughnessy @torgo thanks for your feedback! I formed the event's idea based on lots of questions, concerns, comments, doubts I always get from conference attendees (both web- and mobile devs) after my PWA sessions. And I like to see how it evolves based on the feedback in this repo. Please read another comment about the event format by Charlie Croom from Twitter: https://github.com/webmaxru/progressive-web-conf/issues/3#issuecomment-389274752

Actually, there are 2 directions:

1) to make it more focused on the discussions about standards, roadmaps, different parties intents - to let developers understand what (and when!) to expect in the **future**, to give vendors the community's feedback on some features (like updated way to install the app, etc), and maybe even to "sign" a form of PWA manifesto giving dev community and businesses a long-term vision. That might sound naive, but by that event _I want to give a chance to PWA to survive_, to let the tech giants to align their plans and to find the compromises - all in the name of the dev community.

2) to make it more focused on what's possible to implement **now**, best use-cases, success stories, experience gathered from 3 years of PWA. That's also could be very interesting content (I'm active speaker on that topic myself!)

Combining these 2 approaches? Sounds good for me! But we need a team first (before opening any kind of CFP or any other steps). If someone on this thread has wish and bandwidth to join - let's have a call next week!

**Comment by docluv** [comment]
Looking over the main points listed above feels like it is pretty close to a standardization meeting in the guise of a conference. I don't really see an opportunity to present real-world implementation stories and examples. 
I agree, to me at least, there is a lot of content produced by the Chrome team and some by Edge on getting up and running. But where I find I connect with developers and stakeholders is reviewing how I have solved real implementation issues, like cache invalidation strategies.
Maybe you should add something to this point to your bullet points.

**Comment by webmaxru** [comment]
@docluv what would you like to add/change in p.2 from here https://github.com/webmaxru/progressive-web-conf/issues/2#issuecomment-390192418 ? This point is my message about the wish to have folks from dev community too. Could you find a better wording?
